boards/x86/qemu_x86/qemu_x86_jailhouse_defconfig

```diff
@@ -0,0 +1,22 @@
+CONFIG_X86=y
+CONFIG_SOC_IA32=y
+CONFIG_BOARD_QEMU_X86=y
+CONFIG_CPU_MINUTEIA=y
+CONFIG_IA32_LEGACY_IO_PORTS=y
+CONFIG_JAILHOUSE=y
+CONFIG_JAILHOUSE_X2APIC=y
+CONFIG_KERNEL_ENTRY="__jh_entry"
+CONFIG_PHYS_LOAD_ADDR=0x0
+CONFIG_PHYS_RAM_ADDR=0x0
+CONFIG_IOAPIC_MASK_RTE=n
+CONFIG_PIC_DISABLE=n
+CONFIG_LOAPIC=y
+CONFIG_LOAPIC_TIMER=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_NS16550=y
+CONFIG_UART_CONSOLE=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_XIP=n
+CONFIG_X86_MMU=y
+CONFIG_HW_STACK_PROTECTION=y
```

Comment:
add CONFIG_USERSPACE=y and CONFIG_X86_PAE_MODE=y to match qemu_x86

Reply:
I guess I'll have to move the whole pull request forward to get these. A new pull request, then?

Reply:
not sure what you mean by this, are you saying your branch is out of date? you should be able to just rebase it on master and force-push it

Reply:
I was afraid GitHub was not smart enough to get that. Alright, will do.

Reply:
Now looking at this new DTS config introduction (ddf6a69). How is one suppose to configure that? Only by providing another one from scratch?

Reply:
DTSes don't expect `#ifdef`s  on them and I don't want to mess with the `CONFIG_BOARD` entry, which influences which file extract_dts_includes.py expects :( I'll come back later.